.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+dist: trusty
+language: generic
+
+notifications:
+  email:
+    on_failure: always
+    recipients:
+      - ros@pilz.de
+env:
+  global:
+    - CATKIN_PARALLEL_TEST_JOBS=-p1
+    - ROS_PARALLEL_TEST_JOBS=-j1
```

Comment:
Please try to remove the parallel job reduction (again). It should be possible, to build the packages in parallel, otherwise we should investigate the problem.